### PR TITLE
Update evohome, honeywell markdown with latest changes

### PR DESCRIPTION
**Pull request by zxdavb**
**Description:**
Add zones to evohome platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#18428

Note this is for branch **next**

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by zxdavb** [comment]
@frenck Would you mind kindly reviewing this? - it is the only thing holding up my PR (https://github.com/home-assistant/home-assistant/pull/18428), which has just been merged into dev.

Note that it is for the `next` release of HA as it refers to changes in the `evohome` platform that wont exist until then.

File: source/_components/climate.evohome.markdown [hunk 1]
```diff
@@ -13,15 +13,36 @@ ha_release: 0.80
 ha_iot_class: "Cloud Polling"
 ---
 
-The `evohome` climate platform integrates your _EU-based_ [Honeywell Connect Comfort](https://international.mytotalconnectcomfort.com/Account/Login) CH/DHW controller into Home Assistant, enabling control of its operating mode.
+The `evohome` climate component integrates your _non-US_ [Honeywell Total Connect Comfort (TCC)](https://international.mytotalconnectcomfort.com/Account/Login) system into Home Assistant. It has been tested with the Honeywell evohome multi-zone CH/DHW system, but other systems may also work.
```
Comment (MartinHjelmare) [inline source/_components/climate.evohome.markdown]: This page describes the evohome climate platform. The evohome component is the last modified page in the PR.

```suggestion
The `evohome` climate platform integrates your _non-US_ [Honeywell Total Connect Comfort (TCC)](https://international.mytotalconnectcomfort.com/Account/Login) system into Home Assistant. It has been tested with the Honeywell evohome multi-zone CH/DHW system, but other systems may also work.
```

File: source/_components/climate.evohome.markdown [hunk 2]
```diff
@@ -13,15 +13,36 @@ ha_release: 0.80
...
 
-It is related to the honeywell climate component](/components/climate.honeywell/), which allows limited integration with evohome Heating zones.  These two components should be usuable side-by-side, but YMMV.
+It is distinct from - but related to - the [`honeywell`](/components/climate.honeywell/) climate component, which also allows (limited) integration with Honeywell Connected thermostats; these two components may well be usable side-by-side, but YMMV.
```
Comment (MartinHjelmare) [inline source/_components/climate.evohome.markdown]:
```suggestion
It is distinct from - but related to - the [`honeywell`](/components/climate.honeywell/) climate platform, which also allows (limited) integration with Honeywell Connected thermostats; these two components may well be usable side-by-side, but YMMV.
```

File: source/_components/climate.honeywell.markdown [hunk 3]
```diff
@@ -14,13 +14,15 @@ ha_iot_class: "Cloud Polling"
...
+The `honeywell` climate platform let you control Honeywell Connected thermostats from Home Assistant.
 
 <p class='note'>
-This platform does NOT connect to MyTotalConnectComfort.com.  If you have a Honeywell WIFI thermostat that is connected through MyTotalConnectComfort.com, you may might to take a look at the IFTTT component which can bridge the gap between Home Assistant and MyTotalConnectComfort.com WIFI thermostats on a limited basis.
+This platform is actually a hybrid combination of two distinct climate systems based upon either [somecomfort](https://github.com/kk7ds/somecomfort) (for US-based systems), or [evohome-client](https://github.com/watchforstock/evohome-client) (for international/EU systems) - see the `region` parameter, below. Before you proceed, please be clear which client library is appropriate to your system.
 </p>
 
-To set it up, add the following information to your `configuration.yaml` file:
+Although both regions remain support here, the `evohome-client`-based systems have a new [platform](/components/evohome/) available that may suit your needs better.
```
Comment (MartinHjelmare) [inline source/_components/climate.honeywell.markdown]:
```suggestion
Although both regions remain supported here, the `evohome-client`-based systems have a new [component](/components/evohome/) available that may suit your needs better.
```

File: source/_components/climate.honeywell.markdown [hunk 4]
```diff
@@ -14,13 +14,15 @@ ha_iot_class: "Cloud Polling"
...
+
+To set this climate platform, add the following information to your `configuration.yaml` file:
```
Comment (MartinHjelmare) [inline source/_components/climate.honeywell.markdown]:
```suggestion
To set up this climate platform, add the following information to your `configuration.yaml` file:
```

File: source/_components/climate.honeywell.markdown [hunk 5]
```diff
@@ -30,7 +32,7 @@ climate:
     scan_interval: 600
 ```
 <p class='note'>
-Scan interval is expressed in seconds. Omitting scan_interval may result in too-frequent polling and cause you to rate-limited by Honeywell.
+Scan interval is expressed in seconds. Omitting/mis-configuring `scan_interval` may result in too-frequent polling and cause you to rate-limited by Honeywell.
```
Comment (MartinHjelmare) [inline source/_components/climate.honeywell.markdown]:
```suggestion
Scan interval is expressed in seconds. Omitting or mis-configuring `scan_interval` may result in too-frequent polling and cause you to be rate-limited by Honeywell.
```

File: source/_components/evohome.markdown [hunk 6]
```diff
@@ -13,11 +13,11 @@ ha_release: 0.80
 ha_iot_class: "Cloud Polling"
 ---
 
-The `evohome` platform is the main component to set up and integrate all supported evohome Climate devices.
+The `evohome` platform is the main component to set up and integrate all _non-US_ [Honeywell Total Connect Comfort (TCC)](https://international.mytotalconnectcomfort.com/Account/Login) CH/DHW systems, primarily the Honeywell evohome multi-zone CH/DHW system.  It does not support the home security functionality of TCC.
```
Comment (MartinHjelmare) [inline source/_components/evohome.markdown]:
```suggestion
The `evohome` component is the main component to set up and integrate all _non-US_ [Honeywell Total Connect Comfort (TCC)](https://international.mytotalconnectcomfort.com/Account/Login) CH/DHW systems, primarily the Honeywell evohome multi-zone CH/DHW system.  It does not support the home security functionality of TCC.
```

File: source/_components/evohome.markdown [hunk 7]
```diff
@@ -13,11 +13,11 @@ ha_release: 0.80
...
 
-It uses the [evohomeclient](https://pypi.org/project/evohomeclient/) client library and so will let you control (only) _EU-based_ [Honeywell Connect Comfort](https://international.mytotalconnectcomfort.com/Account/Login) systems.
+It uses v2 of the [evohome-client](https://github.com/watchforstock/evohome-client) client library and so will let you control (only) EU/international systems. It _does not_ leverage the [somecomfort](https://github.com/kk7ds/somecomfort) client library as used by US-based systems; for those, you may find what you need at the [honeywell climate component](/components/climate.honeywell/).
```
Comment (MartinHjelmare) [inline source/_components/evohome.markdown]:
```suggestion
It uses v2 of the [evohome-client](https://github.com/watchforstock/evohome-client) client library and so will let you control (only) EU/international systems. It _does not_ leverage the [somecomfort](https://github.com/kk7ds/somecomfort) client library as used by US-based systems; for those, you may find what you need at the [honeywell climate platform](/components/climate.honeywell/).
```

File: source/_components/evohome.markdown [hunk 8]
```diff
@@ -13,11 +13,11 @@ ha_release: 0.80
...
 
-Currently, only Controllers and Zones are supported; support for DHW controllers will be added at a later time. More information is available [evohome climate component](/components/climate.honeywell/) page.
+Currently, only Controllers and Heating zones are supported; support for DHW controllers will be added at a later time. More information is available on the [evohome climate component](/components/climate.evohome/) page.
```
Comment (MartinHjelmare) [inline source/_components/evohome.markdown]:
```suggestion
Currently, only Controllers and Heating zones are supported; support for DHW controllers will be added at a later time. More information is available on the [evohome climate platform](/components/climate.evohome/) page.
```

**Comment by zxdavb** [comment]
Clearly, I still haven't got my head around the nomenclature, and the difference between a platform & a component!

Thanks for your help.

**Review by MartinHjelmare — APPROVED** [review]
Can be merged when build passes.

**Pull request merged**